### PR TITLE
 Leslie Mei Homework 1

### PR DESCRIPTION
**Pull request by lesliesmei**
Using GitHub Desktop, I downloaded the master file for hw01. Then I right-clicked to find the file in Finder so that I could edit the correct file. I found that the most challenging part of this assignment was figuring out how the GitHub system worked - at first, I couldn't get the edits to show up because I had downloaded hw01 not through the GitHub Desktop app and edited a different version of the file in my Downloads folder. The process made a lot more sense once I thought of it in terms of syncing files with a Cloud.

Edit: I also had trouble pushing the homework because I cloned from the master copy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # hw01
 Repository for homework 01
```
Comment (tiffany-wu) [inline README.md:2]: Can delete this in the final draft.

**Review by tiffany-wu — COMMENTED** [review]
Evaluation HW-01
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   x        |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Leslie met the core requirements of this assignment by demonstrating 4 of the Markdown elements listed.
* However, I wish she added an image and/or elaborated on her facts so we get to know more about her!
* This is not really related to the code at all, but _The Brothers Karamazov_ is also one of my all-time favorite books!! Let's get together and gush about Alyosha and/or Ivan some time.

**Comment by risaman** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Coding is good and implements a variety of features in markdown 
* Could be improved with a picture and/or a bit more bio information

**Comment by alexanme** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |        x      |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Appropriately uses several of md's formatting features
* Consistent and concise!! But maybe a little too concise, wish there was more info :(
* I wanna see a picture of smthg! anything!

**Comment by chandlerjames** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |      X     |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* Great  job Leslie, this is excellent code
* All criterion were met in your code
* Would have learned to love more about _The Brothers Karamazov_

**Comment by mgneely** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |X           |              |            |
| **Coding strategy**         |X           |              |            |
| **Presentation**            |X           |              |            |
| **Achievement, creativity** |           |X              |            |
| **Ease of access**          |X           |              |            |

Remarks:

* Coding is very easy to follow! 
* Wish you used a few more of the more creative/difficult functions, such as sub bullet, sub header, picture etc.
* Met requirements of including 4 markdown functions

**Comment by mayank-kgp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |          |       x       |            |
| **Ease of access**          |        x  |              |            |

Happy to see that you learned the GitHub workflow and used markdown without any error.

**Pull request closed without merging**